app/src/androidTest/java/com/example/mindyourpet/PetDatabaseTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.example.mindyourpet
+
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.example.mindyourpet.database.Pet
+import com.example.mindyourpet.database.PetDatabase
+import com.example.mindyourpet.database.PetDatabaseDao
+import com.example.mindyourpet.database.Reminder
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+import kotlin.jvm.Throws
+
+
+/**
+ * This is not meant to be a full set of tests. For simplicity, most of your samples do not
+ * include tests. However, when building the Room, it is helpful to make sure it works before
+ * adding the UI.
+ */
+
+@RunWith(AndroidJUnit4::class)
+class PetDatabaseTest {
+
+    private lateinit var petDao: PetDatabaseDao
+    private lateinit var db: PetDatabase
+
+    @Before
+    fun createDb() {
+        val context = InstrumentationRegistry.getInstrumentation().targetContext
+        // Using an in-memory database because the information stored here disappears when the
+        // process is killed.
+        db = Room.inMemoryDatabaseBuilder(context, PetDatabase::class.java)
+            // Allowing main thread queries, just for testing.
+            .allowMainThreadQueries()
+            .build()
+        petDao = db.PetDatabaseDao
+
+        initializeDatabase()
+    }
+    fun initializeDatabase(){
+
+        val p = Pet(petId = 0,name="Bingo", speciesId = 0)
+        val p1=Pet(petId = 1,name="Jeans",speciesId = 1)
+        val p2=Pet(petId = 2,name="Kim", speciesId = 4)
+        val p3 = Pet(petId = 3,name="Bob",speciesId = 3)
+        val p4 = Pet(petId = 4,name="Burro", speciesId = 2)
+        val p5 = Pet(petId = 5,name="Elvert",speciesId =  0)
+
```

Comment:
You should never have two blank lines within a method. Please go through your code and consider whether each blank line belongs.